packages/@romejs/js-compiler/lint/rules/react/noUnsafe.ts

```diff
@@ -0,0 +1,55 @@
+import {Path, TransformExitResult} from "@romejs/js-compiler";
+import {descriptions} from "@romejs/diagnostics";
+import {AnyNode, JSIdentifier} from "@romejs/ast";
+
+const UNSAFE_METHODS = [
+	{
+		oldMethod: "UNSAFE_componentWillMount",
+		newMethod: "componentDidMount",
+		details: "See https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html.",
```

Comment:
```suggestion
		details: "See <hyperlink target="https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html" />.",
```

will allow these to be clickable in the terminal.